### PR TITLE
update tests for transformPropsUtil

### PR DESCRIPTION
**Pull request by jansule**
### SUMMARY

Updates test for `transformPropsUtil`.

### CODE REVIEW

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts [hunk 1]
```diff
@@ -17,124 +17,147 @@
...
+      expect(countGeometries).toEqual(2);
+    });
 
-  it('groups in the correct count of geometries', () => {
-    const countOfGeometries = Object.keys(result).length;
-    expect(countOfGeometries).toEqual(2);
+    it('removes the geometry from the data field for grouped data', () => {
+      const result = groupByLocationTs(groupedTimeseriesChartData);
+
+      expect(result).toEqual(
```
Comment (JakobMiksch) [inline superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts:51]: could you write a short comment what is happening here?

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts [hunk 2]
```diff
@@ -17,124 +17,147 @@
...
+      const result = getEchartConfigs(
+        nonTimeSeriesChartData,
+        undefined,
+        pieChartConfig,
+        geomColumn,
+        pieChartProps,
+        chartTransformer,
+      );
+
+      expect(result).toEqual(
```
Comment (JakobMiksch) [inline superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts:120]: short comment what is happening here would be great

**Review by JakobMiksch — APPROVED** [review]
LGTM and test also runs for me. 
Only two questions inline that I find not easy to understand on first sight.

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts (hunk 1 above)
Comment (jansule) [reply superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts:51]: It's a way of checking only the relevant parts of a complex structure. This way, we do not have to provide all keys of an object, or all items of an array, but only those that we actually want to check.

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts (hunk 2 above)
Comment (jansule) [reply superset-frontend/plugins/plugin-chart-cartodiagram/test/util/transformPropsUtil.test.ts:120]: See comment above

**Pull request merged**